File: test/rust-hello/handler.rs
use serde_json::{json, Value};

pub fn handler(event: Value) -> Value {
    json!({
        "status": 200,
        "headers": { "Content-Type": "application/json" },
        "body": json!({
            "message": "Hello from Rust!",
            "runtime": "rust"
        }).to_string()
    })
}

File: tests/fixtures/local-dev-samples-migrated/php-hello/handler.php
<?php

function handler(array $event): array {
    return [
        'status' => 200,
        'headers' => [
            'Content-Type' => 'application/json',
        ],
        'body' => json_encode([
            'message' => 'Hello from PHP!',
            'runtime' => 'php'
        ]),
    ];
}
